composer.json

```diff
@@ -10,8 +10,8 @@
         "swarrot/swarrot": "~2.0|~3.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "~1.12",
-        "phpunit/phpunit": "^5.7"
+        "friendsofphp/php-cs-fixer": "^1.12|^2.0",
+        "phpunit/phpunit": "^5.7|^6.0|^7.0"
```

Comment:
```suggestion
        "phpunit/phpunit": "^7.0"
```

Reply:
Should I drop support for php 5.6 to 7.0 ? (see  .travis.yml )

Reply:
We'll do this in another PR, forget my suggested changes!

We'll need:
* to remove support of PHP 5.6 / 7.0
* to bump all requirements (including non-dev)

if we add strict casting: to release a new major version.

I merge your PR right now, feel free to open another one if you want to remove php 5.6 / 7.0 support.